EIPS/eip-1962.md

```diff
@@ -0,0 +1,224 @@
+---
+eip: 1962
+title: EC arithmetic and pairings with runtime definitions
+author: Alex Vlasov (@shamatar)
+discussions-to: https://github.com/ethereum/EIPs/pull/1962
```

Comment:
This cannot point to a pull request according to EIP-1 rules.

The recommended options are a github issue on this repo or a forum on ethereum-magicians.org. Since the EIP1829 discussion is on ethereum-magicians.org, perhaps that is the better forum.

Reply:
Added